webapi/src/main/scala/org/knora/webapi/responders/v2/ListsResponderV2.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright © 2015-2018 the contributors (see Contributors.md).
+ *
+ * This file is part of Knora.
+ *
+ * Knora is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Knora is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.knora.webapi.responders.v2
+
+import akka.actor.Status
+import akka.pattern._
+import akka.stream.ActorMaterializer
+import org.knora.webapi.IRI
+import org.knora.webapi.messages.admin.responder.listsmessages.{ListGetRequestADM, ListGetResponseADM, ListNodeInfoGetRequestADM, ListNodeInfoGetResponseADM}
+import org.knora.webapi.messages.admin.responder.usersmessages.UserADM
+import org.knora.webapi.messages.v2.responder.listsmessages._
+import org.knora.webapi.responders.Responder
+import org.knora.webapi.util.ActorUtil.{future2Message, handleUnexpectedMessage}
+
+import scala.concurrent.Future
+
+/**
+  * Responds to requests relating to the creation of mappings from XML elements and attributes to standoff classes and properties.
```

Comment:
This class comment isn't correct.

Reply:
done in https://github.com/dhlab-basel/Knora/pull/852/commits/15af2141af9d76c2335f338db9f936e289517da7